program-runtime/src/invoke_context.rs

```diff
@@ -719,19 +720,28 @@ impl<'a> InvokeContext<'a> {
         };
 
         for entry in self.builtin_programs.vec.iter() {
-            if entry.program_id == builtin_id {
+            if entry.0 == builtin_id {
+                const ENTRYPOINT_KEY: u32 = 0x71E3CF81;
```

Comment:
What's the significance of this value? `0x71E3CF81`

Reply:
It is the hash of the string "entrypoint":
https://github.com/solana-labs/rbpf/blob/e9c2d206e18a5e759b63f39353b029dec6dc4247/src/ebpf.rs#L579

Reply:
Should we add a comment to reflect that?